packages/babel-plugin-transform-runtime/test/fixtures/runtime-corejs2/regenerator-runtime-old-runtime-version/output.js

```diff
@@ -1,7 +1,7 @@
 var _regeneratorRuntime = require("@babel/runtime-corejs2/regenerator");
 
-void /*#__PURE__*/_regeneratorRuntime.mark(function _callee() {
-  return _regeneratorRuntime.wrap(function _callee$(_context) {
+void /*#__PURE__*/(() => _regeneratorRuntime)().mark(function _callee() {
```

Comment:
Do we need to mark the new IIFE as pure, too?

cc @devongovett

Reply:
It's actually marking the `.mark()` call (tested in terser, I was surprised that it works).